Gemfile.template

```diff
@@ -30,3 +30,4 @@ gem "rack-test", :require => "rack/test", :group => :development
 gem "rspec", "~> 3.5", :group => :development
 gem "webmock", "~> 3", :group => :development
 gem "jar-dependencies", "= 0.4.1" # Gem::LoadError with jar-dependencies 0.4.2
+gem "csv", "~> 3" # Bundled version of CSV with jruby >=9.3.0.0 < 9.3.8.0 has a thread leak
```

Comment:
Not familiar with gem template but it seems we need to upgrade from `3.1.2` to `3.2.5`. Shouldn't be `gem "csv", "~> 3.2"`?

Reply:
The intent was to be consistent with the other imports in our Gemfile, where we use the [pessimistic version constraint](https://guides.rubygems.org/patterns/#pessimistic-version-constraint) to constrain dependencies to the same major version, so specifying `~> 3` will bring in `3.2.5` as the most recent version of v3, and we typically expect our imported gems to follow semantic versioning, and not to have backwards incompatible changes.